python/lsst/dax/apdb/cassandra/apdbCassandra.py

```diff
@@ -565,10 +578,42 @@ def list_databases(cls, host: str) -> Iterable[str]:
         result = session.execute(query, (table_name,))
         keyspaces = [row[0] for row in result.all()]
 
+        infos = {keyspace: DatabaseInfo(name=keyspace) for keyspace in keyspaces}
+        if keyspaces:
+            # Retrieve roles for each keyspace.
+            template = ", ".join(["%s"] * len(keyspaces))
+            query = (
+                "SELECT resource, role, permissions FROM system_auth.role_permissions "
+                f"WHERE resource IN ({template}) ALLOW FILTERING"
+            )
+            resources = [f"data/{keyspace}" for keyspace in keyspaces]
+            try:
+                result = session.execute(query, resources)
+                for row in result:
+                    _, _, keyspace = row[0].partition("/")
```

Comment:
I'm not familiar with Cassandra. Is it safe to assume that `resource` has exactly two slashes?

Reply:
`resource` will have one slash. It's `partition`, not `split`, and it always returns 3 items, I only need what comes after a slash.